mscore/resourceManager.h

```diff
@@ -24,26 +27,41 @@ class ResourceManager : public QDialog, public Ui::Resource
 
     virtual void hideEvent(QHideEvent*);
     QByteArray txt;
-    void displayLanguages();
-    void displayExtensions();
+    static QThreadPool workerThreads;
     bool verifyFile(QString path, QString hash);
     bool verifyLanguageFile(QString filename, QString hash);
+    void refreshPluginButton(int row, bool updated = true);
 
 public:
     explicit ResourceManager(QWidget *parent = 0);
     void selectLanguagesTab();
     void selectExtensionsTab();
 
     static inline QString baseAddr() { return "http://extensions.musescore.org/3.3/"; }
+    static inline QString pluginRepoAddr() { return "https://musescore.org/en/plugins"; }
```

Comment:
Not 100% sure, but it may be possible to drop the `/en` part.

Reply:
In the MuseScore application, we have no user info(cookies or sth else) in that http request, so I believe it always redirects to `/en/plugins`. And redirection may mean more loading time ;)
![image](https://user-images.githubusercontent.com/28099570/63313545-7086a280-c337-11e9-8c43-6f61baab0506.png)
![image](https://user-images.githubusercontent.com/28099570/63313550-754b5680-c337-11e9-9e48-fa9223a3d444.png)

Reply:
OK, some 300ms saved ;-)